pipelines/docker.go

```diff
@@ -48,7 +48,7 @@ func GrafanaImageTags(base BaseImage, registry string, opts TarFileOpts) []strin
 		version += "-ubuntu"
 	}
 
-	if opts.Distro != "" && opts.Distro != "linux/amd64" {
```

Comment:
This is done because following https://drone.grafana.net/grafana/grafana-private-mirror/7532/1/6 the -amd64 is expected on image-tags.